spec/controllers/api/events_controller_spec.rb

```diff
@@ -87,7 +87,7 @@ def overridden_params(new_params)
 
         it "should return 422" do
           allow(subject).to receive(:create_params)
-            .and_raise(ActionDispatch::ParamsParser::ParseError.new('test', 'test'))
+            .and_raise(JsonSchema::ValidationError.new)
```

Comment:
Not blocking - i think this whole `event_controller.rb` can go - it was originally made to integrate with our old API events to keep counts up to date for existing Ouroboros projects. 

now that isn't running anymore we can remove this code. 

perhaps a follow up PR or add to a task pile for the rails 5.0 upgrade.